packages/apps/proof-of-us/src/components/UserLayout/UserLayout.tsx

```diff
@@ -19,7 +19,7 @@ const UserLayout: FC<PropsWithChildren> = ({ children }) => {
 
   return (
     <motion.div
-      style={{ position: 'absolute', width: '100%' }}
+      style={{ position: 'relative', width: '100%' }}
```

Comment:
this needs to be on absolute. otherwise the router layout animations will not work correctly